client-v2/src/components/EditionFeedSectionHeader.tsx

```diff
@@ -44,14 +49,14 @@ class EditionFeedSectionHeader extends React.Component<ComponentProps> {
 
     return (
       <>
-        <Link to="/v2/manage-editions/daily-edition">
```

Comment:
This is because the react app's route is `/v2`, thus all paths are relative to it.

Reply:
it'd be safer to parameterise this -- it's available in constants/urls.ts as appRoot 👍

Reply:
👍 done.